website/static/css/custom.css

```
@@ -0,0 +1,12 @@
+/* Custom styles */
+
+div.warning   {
+    background: #cc2020;
+    padding: 6pt;
+    border-radius: 6pt;
+}
+
+div.warning > h3, div.warning > h4 {
+    color: #fff;
+    font-weight: lighter;
+}
```

```suggestion
.warning {
    background: #cc2020;
    padding: 6pt;
    border-radius: 6pt;
    color: #fff;
}

.warning > h3,
.warning > h4 {
    font-weight: lighter;
} 
```

- You can put `color` on the parent element and that'll be the default for the parent.
- I'd strip the `div` requirement and just base it on the class (`.warning`).

---

Boo, there's a color override for the header styles in place:
```css
.mainContainer h1, .mainContainer h2, .mainContainer h3, .mainContainer h4, .mainContainer h5, .mainContainer h6 {
    color: #1a1a1a;
}
```
So yeah, the above won't work without overriding that.

---

```suggestion
.warning {
    background: #cc2020;
    padding: 6pt;
    border-radius: 6pt;
    color: #fff;
}

.warning > h3,
.warning > h4 {
    font-weight: lighter;
    color: #fff;
} 
```
By having it in both you cover the case of someone putting text inside the red box without being a header. The other header types are left unaddressed, but /shrug.

---

```suggestion
.warning   {
    background: #cc2020;
    padding: 4pt;
    border-radius: 6pt;
    color: #fff;
}

div.warning > h3, 
div.warning > h4   {
    color: #fff;
    font-weight: normal;
}
```